### DISCUSSION TITLE
Rename master?

### DISCUSSION THREAD

**Discussion by alflanagan** [pulsar-edit/.github#46] [General]
This is probably not a high-priority item, but we should create a branch `main` and make it the main trunk. There's no point in keeping a branch name that alienates people.

**Comment by mauricioszabo** [comment]
We discussed a little bit about this on Discord, but for now we're going to keep master as the trunk - there's simply too much on the codebase we don't know or understand, and changing anything may (and in fact is) cause bugs we don't know where to start.

As for the name, think about `master` as "master copy" (for media) or something else. There are multiple meanings of the word (like master's degree, master Yoda, etc) and most are not related to anything bad, really...

**Comment by confused-Techie** [comment]
Although that isn't to say it's not on our agenda at all. Since it is an issue parts of the community are concerned about we do want to address it. But like the points made above there's to much risk to break to much, at a time where we are just now understanding and fixing the codebase.

But I'd imagine once we have a stable, updated, and understood codebase then we can start making changes like this.

**Comment by kaosine** [comment]
I'm going to echo mauricio that it's hard to do this with what we don't know about the codebase having inherited it off our original upstream partners. So it's hard to do this unless someone knows what they're doing to re-base and reconfigure it. Personally I'd like to see that change just for the sake of inclusivity despite it not having that connotation in other languages. It's just a sanity fix imo but I think we're outnumbered here on this one tbh by those opposing it for various reasons.
